Foundation.json

```diff
@@ -1,20 +1,6 @@
 {
-    "driver": {
```

Comment:
1. 'simpixel' is the default driver.
2. More flexible to use the `--dim=12` flag.